README.md

```diff
@@ -3,3 +3,18 @@ This is the code of the GitHub Pages site containing documentation for Elastic P
 The GitHub Pages site can be found at:
 
 https://elasticpath.github.io/react-pwa-reference-storefront/
+
+
+### To Set Up Documentation Locally for Development Purposes
+*Prerequisites: Install Xcode and Command Line Tools*
+
+1. Install the latest version of rvm:
+`\curl -sSL https://get.rvm.io | bash -s stable`
+2. Install the latest version of ruby with rvm:
+`rvm install ruby --latest`
+3. Run `gem install github-pages`
+4. Run `gem install bundler jekyll`
+5. Run `bundle install`
+6. Run `jekyll build`
+7. Run `jekyll serve`
+The documentation is now available locally at `http://127.0.0.1:4000/react-pwa-reference-storefront/`
```

Comment:
The documentation appears at `http://127.0.0.1:4000/react-pwa-reference-storefront/` locally.

Reply:
I modified this a bit.  I felt looking at it the word locally got lost so I put it after appears and then the link is the last of the sentence.